src/core/migrations/0074_auto_20220721_1448.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2022-07-21 12:35
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.utils import translation
+from django.conf import settings as django_settings
+
+
+OLD_DEFAULT_VALUE = 'Dear {{ review_assignment.reviewer.full_name }},<br/><br/>We are requesting that you undertake a review of "{{ article.title }}" in {{ article.journal.name }}.<br/><br/>We would be most grateful for your time as the feedback from our reviewers is of the utmost importance to our editorial decision-making processes.<br/><br/>You can let us know your decision or decline to undertake the review: {{ review_url }} <br/><br/>{{ article_details }}<br/><br/>Regards,<br/>{{ request.user.signature|safe }}'
+VARIANT_ONE = 'Dear {{ review_assignment.reviewer.full_name }},<br/><br/>We are requesting that you undertake a review of "{{ article.title }}" in {{ article.journal.name }}.<br/><br/>We would be most grateful for your time as the feedback from our reviewers is of the utmost importance to our editorial decision making processes.<br/><br/>You can let us know your decision or decline to undertake the review: {{ review_url }} <br/><br/>Regards,<br/>{{ request.user.signature|safe }}<br/>{{ article_details }}'
+VARIANT_TWO = 'Dear {{ review_assignment.reviewer.full_name }},<br/><br/>We are requesting that you undertake a review of "{{ article.title }}" in {{ article.journal.name }}.<br/><br/>We would be most grateful for your time as the feedback from our reviewers is of the utmost importance to our editorial decision making processes.<br/><br/>You can let us know your decision or decline to undertake the review: {{ review_url }} <br/><br/>Regards,<br/>{{ request.user.signature }}<br/>{{ article_details }}'
+
+NEW_VALUE = 'Dear {{ review_assignment.reviewer.full_name }},<br/><br/>We are requesting that you undertake a review of "{{ article.title }}" in {{ article.journal.name }}.<br/><br/>We would be most grateful for your time as the feedback from our reviewers is of the utmost importance to our editorial decision-making processes.<br/><br/>You can let us know your decision or decline to undertake the review: {{ review_url }} <br/><br/> This review assignment is due on {{ review_assignment.date_due|date:"Y-m-d" }}.  <br/><br/>{{ article_details }}<br/><br/>Regards,<br/>{{ request.user.signature|safe }}'
+
+
+def update_default_setting(apps, schema_editor):
+    """
+    Updates the review_assignment setting for a journal where it has not been edited.
+    """
+    with translation.override(django_settings.LANGUAGE_CODE):
+        SettingValue = apps.get_model('core', 'SettingValue')
+        setting_value = SettingValue.objects.get(
+            setting__name='review_assignment',
+            journal=None,
+        )
+        language_var = "value_{}".format(django_settings.LANGUAGE_CODE)
+        setattr(setting_value, language_var, NEW_VALUE)
```

Comment:
I may not be reading this right, but it would seem to save NEW_VALUE and tag it as French for an installation that was set to French. Wouldn't we want to avoid that and only change English values?

Reply:
Specifically for the default value. At the moment default values are only loaded in English. Still, may be for the best to set this to EN.

Reply:
OK, let's set this to EN then.